File: pset6/mario/more/mario.py
# Get the input
a = 0
while a < 1:
    n = input("Height: ")
    if n >= 1 and n <= 8:
        a += 1

#
a = 0
b = 0
c = 0
space = n - (a + 1)

for a in range(0, n):
    for b in range(0, space):
        print(" ", end="")

    space = space - 1

    for b in range(0, a + 1):
        print("#", end="")

    print("  ", end="")

    for c in range(0, a + 1):
        print("#", end="")

    print("\n", end="")

File: pset1/mario/more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Checkimg input
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    // Create # table
    for (int a = 1; a <= n; a++)
    {
        for (int b = n; b > a; b--)
        {
            printf(" ");
        }
        for (int c = 1; c <= a; c++ )
        {
            printf("#");
        }
        printf("  ");
        for (int d = 1; d <= a; d++)
        {
            printf("#");
        }
        printf("\n");
    }
    return 0;
}
